modules/virt-creating-a-custom-image-boot-source-web.adoc

```diff
@@ -0,0 +1,35 @@
+// Module included in the following assemblies:
+//
+// * virt/virtual_machines/virtual_disks/virt-creating-and-using-boot-sources.adoc
+
+[id="virt-creating-a-custom-disk-image-boot-source-web_{context}"]
+= Creating a custom boot source
+
+Sometimes you want to prepare a custom disk image, based on an existing disk image, for use as a boot source.
+
+This procedure guides you through:
+
+* Preparing a custom disk image.
+* Creating a boot source from the custom disk image.
+* Attaching the boot source to a custom template.
+
+.Procedure
+. In the {VirtProductName} console, click *Workloads* > *Virtualization* from the side menu.
+. Click the *Templates* tab.
+. Click the link in the *Source provider* column for the template you want to customize. A window displays, indicating that the template currently has a *defined* _source_.
+. In the window, click the *Customize source* link.
+. In the *About boot source customization* window, click *Continue* to proceed with customization, after reading the information provided about the boot source customization process.
```

Comment:
Maybe reorder the sentence so the "reading" part is first:

```
. In the *About boot source customization* window, read the information provided about the boot source customization process, and then click *Continue* to proceed with customization.
```

Reply:
Also in general, I didn't point it out in the various other places that were similar, but something to consider is what IBM SG says about "then (_adverb_)" in the "Word usage" glossary, regarding two-step procedures written as one sentence. Technically, they should be joined by ", and then..." :shrug:

Reply:
Reworded and added the "thens" in other places where needed.